src/datanode/src/server/grpc/ddl.rs

```diff
@@ -72,7 +75,12 @@ impl Instance {
 
         let request = create_expr_to_request(table_id, expr).context(CreateExprToRequestSnafu);
         let result = futures::future::ready(request)
-            .and_then(|request| self.sql_handler().execute(SqlRequest::CreateTable(request)))
+            .and_then(|request| {
```

Comment:
I think processing DDL statements may need to use the session too( accessing session variables, for example), even though there is only a schema name in the session right now. Can we pass the session context to these DDL handlers too?

Reply:
because our grpc service is going to be rewritten, I'm not handling any "use" database in grpc related codes in this PR. maybe part 2 or 3